### ISSUE TITLE
[Question] Is there an option to include private methods in the documentation?

### ISSUE THREAD

**Issue by wullink-ps** [dotnet/docfx#836]
Probably a bit of an silly question but: currently we are trying to use docfx for internal documentation, in which scenario it sometimes makes sense to include xml comments on private methods. Is there currently an option to do something like that?

Sorry this is probably not the right place to ask such questions, but wasn't sure where to ask otherwise.

**Comment by vwxyzh** [comment]
No option is available by now.
You can modify [filters](https://github.com/dotnet/docfx/blob/dev/src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs#L45) for your purpose.

**Comment by kierancoulter** [comment]
@vwxyzh does this require modifying DocFX source? I have tried to do this while referencing
https://dotnet.github.io/docfx/tutorial/howto_filter_out_unwanted_apis_attributes.html

I tried by creating an [InternalDoc] attribute and adding this attribute to my private methods for which I want to expose documentation. I then made a filterConfig.yml containing:

apiRules:
- include:
    hasAttribute:
      uid: <namespace>.InternalDocAttribute

However this is not working. I have tried including ctorArguments as well.
